index.html

```diff
@@ -75,15 +75,49 @@ <h1>Discover stays<br>
                     </div>
                     <div class="filter-block col-sm-12">
                         <div class="filter-label">
-                            <label>2 Adults — 0 Children — 1 Room</label>
+                            <label  id="lable_value">2 Adults — 0 Children — 1 Room</label>
                         </div>
                         <div class="filter-input">
-                            <input type="text" name="filter-abults" placeholder="2 Adults">
+                            <input id="filter-input__adults" type="text" name="filter-adults" value="2 Adults">
```

Comment:
All id's can be named without BEM. Use simple kebab-case. For example, `filter-adults`